src/main/java/com/neuvector/Scanner.java

```diff
@@ -65,7 +65,7 @@ public static ScanRepoReportData scanRegistry(Registry registry, NVScanner nvSca
                 .withUserAndGroup(getDockerUserGroupCmdArg(getScanReportPath(nvScanner.getNvMountPath())))
                 .withName(generateScannerName())
                 .withVolume(Scanner.SOCKET_MAPPING)
-                .withVolume(getMountPath(nvScanner))
+                .withVolume(appendSELinuxSuffixIfRequired(nvScanner.isSELinuxSuffixRequired(), getMountPath(nvScanner)))
```

Comment:
Similar to my comments in `insight-scanner`.. Something similar to:

```
getMounthPath(nvScanner, bindMountShared)
```

might be a bit better from naming/readability perspective.

Reply:
Line 131 needs to be modified too, isn't it?

Reply:
Commit: [3faba33](https://github.com/neuvector/scanner-java-wrapper/pull/11/commits/3faba3302567008c08fd51c367f25a0d2d5553a8)